lib/features/composer/presentation/widgets/recipient_tag_item_widget.dart

```diff
@@ -82,9 +82,9 @@ class RecipientTagItemWidget extends StatelessWidget {
                         deleteIcon: SvgPicture.asset(_imagePaths.icClose, fit: BoxFit.fill),
                         labelStyle: RecipientTagItemWidgetStyle.labelTextStyle,
                         backgroundColor: _getTagBackgroundColor(),
-                        shape: RoundedRectangleBorder(
-                          borderRadius: const BorderRadius.all(Radius.circular(RecipientTagItemWidgetStyle.radius)),
-                          side: _getTagBorderSide(),
+                        side: _getTagBorderSide(),
```

Comment:
It has any impaction for mobile app?

Reply:
It does not affect the mobile app. I only add `width` for the border and to switch to `const` widget for `RoundedRectangleBorder` use the `borderSide` of the outer widget instead of using it within it.

![Simulator Screenshot - iPhone 15 Pro Max - 2024-02-02 at 18 15 14](https://github.com/linagora/tmail-flutter/assets/80730648/8f677e7b-7b9c-4c15-b580-c0178bdb56ad)